test/README.md

```diff
@@ -0,0 +1,111 @@
+# Test Files for TypeScript/React Support PR
```

Comment:
NIT: These test files could be in a subdir, such as `test/typescript/`. That way we have room to add testing instructions for other providers.